src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -155,6 +155,15 @@ def _txt2obj_gc(backend, name):
     return obj
 
 
+def _encode_ocsp_nocheck(backend, ext):
+    """
+    The OCSP No Check extension is defined as a null ASN.1 value. We can just
+    return that value directly here in the pp, r tuple form the other
+    extension encoding functions use.
```

Comment:
I'm not sure I understand this comment, since it is still returning a tuple.

Does OpenSSL not have a function that does this, I feel like it would contribute to the readability.